src/main/java/it/gov/pagopa/admissibility/utils/Utils.java

```diff
@@ -41,4 +46,48 @@ public static <T> T getHeaderValue(Message<?> message, String headerName) {
     public static Long euro2Cents(BigDecimal euro){
         return euro == null? null : euro.multiply(ONE_HUNDRED).longValue();
     }
+
+    //region birthdate from fiscalcode
+    public static LocalDate calculateBirthDateFromFiscalCode(String fiscalCode) {
+        String fiscalCodeStructureRegex = "^([A-Za-z]{6}[0-9lmnpqrstuvLMNPQRSTUV]{2}[abcdehlmprstABCDEHLMPRST]{1}[0-9lmnpqrstuvLMNPQRSTUV]{2}[A-Za-z]{1}[0-9lmnpqrstuvLMNPQRSTUV]{3}[A-Za-z]{1})$";
+        if (!fiscalCode.matches(fiscalCodeStructureRegex)) {
+            throw new IllegalArgumentException("[ADMISSIBILITY] Fiscal code is not valid!");
+        }
+
+        // Extract birthdate characters from the fiscal code
+        String birthDateCode = fiscalCode.substring(6, 11);
+
+        // Extract birth year, month, and day from the code
+        int birthYearDigits = Integer.parseInt(birthDateCode.substring(0, 2));
```

Comment:
in order to handle the omocode useCase, we could, using a common method which accept just the 2 character string, take into account it in order to transform the string in a integer
see: https://fiscomania.com/omocodia/

we should perform the following replace before to parseInt:
0=L | 1=M | 2=N | 3=P | 4=Q
5=R | 6=S | 7=T | 8=U | 9=V